h2o-core/src/main/java/hex/Model.java

```diff
@@ -1518,7 +1518,7 @@ public boolean testJavaScoring(Frame data, Frame model_predictions, double rel_e
               System.err.println("Row: " + rowData.toString());
             }
           }
-          totalMiss = miss;
+          totalMiss += miss;
```

Comment:
This bug would have masked a failing POJO double[] if the EasyPredict API had passed. Unlikely, but possible.